scripts/init_contract.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# This command is used to automate the store and init process of
+# smart contracts. Examples:
+
+# Without params: sh scripts/migrate.sh ./artifacts/financial_snapshot_contract.wasm
+# With init params: sh scripts/migrate.sh ./artifacts/account_history_contract.wasm '{"limit": 10, "value_denom": "uusdc", "expiration": {"at_time": "604800000000000"}}'
+# Common extra aguments: sh scripts/migrate.sh ./artifacts/financial_snapshot_contract.wasm {} '--node test.com'
```

Comment:
@Wgil thanks for those changes, you might wanna update the examples here as well as the file name changed, thank you

Reply:
Thanks